src/main/java/github/scarsz/discordsrv/listeners/DiscordConsoleListener.java

```diff
@@ -82,7 +84,14 @@ public void onGuildMessageReceived(GuildMessageReceivedEvent event) {
 
         // log command to console log file, if this fails the command is not executed for safety reasons unless this is turned off
         try {
-            String fileName = DiscordSRV.config().getString("DiscordConsoleChannelUsageLog");
+            DateTimeFormatter dtf = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+            LocalDateTime now = LocalDateTime.now();
+            File dataFolder = DiscordSRV.getPlugin().getDataFolder();
+            if (!dataFolder.exists()) dataFolder.mkdir();
+            File logFolder = new File(dataFolder.getAbsolutePath() + File.separator + "logs");
+            if (!logFolder.exists()) logFolder.mkdir();
+            String fileName = logFolder + File.separator + "console-log-" + dtf.format(now) + ".log";
```

Comment:
Change this to `File file = new File(logFolder, "console-log-" + dtf.format(now) + ".log");`